components/google-cloud/google_cloud_pipeline_components/experimental/remote/gcp_launcher/upload_model_remote_runner.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2021 The Kubeflow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import time
+from os import path
+from google_cloud_pipeline_components.experimental.proto.gcp_resources_pb2 import GcpResources
+from google.protobuf import json_format
+from .utils import artifact_util
+from .utils import json_util
+import requests
+import google.auth
+import google.auth.transport.requests
+
+_POLLING_INTERVAL_IN_SECONDS = 20
```

Comment:
not blocking -  I think we mostly have been using a 30 sec interval. should we use a central config for this instead of each component designing their own.

Reply:
yeah missed this comment. i'll reuse this in a follow up PR, after the refactoring from @KevinBNaughton